schemas/expconf/v0/experiment.json

```diff
@@ -71,13 +71,9 @@
         "entrypoint": {
             "type": [
                 "string",
+                "array",
```

Comment:
Oh, sorry, I left the comment about "items" in _gen.py, but you need to make the change here, not in _gen.py (which is generated from this)